lib/livebook/teams/deployment_group.ex

```diff
@@ -23,7 +24,7 @@ defmodule Livebook.Teams.DeploymentGroup do
   def changeset(deployment_group, attrs \\ %{}) do
     changeset =
       deployment_group
-      |> cast(attrs, [:id, :name, :mode, :hub_id, :clustering, :zta_provider, :zta_key])
+      |> cast(attrs, [:id, :name, :mode, :hub_id, :clustering, :zta_provider, :zta_key, :url])
```

Comment:
We should most likely port and bring the validation URLs here too, for faster feedback:

```
|> update_change(:url, fn url ->
      if url do
        url
        |> String.trim_leading("http://")
        |> String.trim_leading("https://")
        |> String.trim_trailing("/")
      end
    end)
    |> validate_format(:url, ~r/(^[^\/\.\s]+(\.[^\/\.\s]+)+)(\/[^\s]+)?$/,
      message: "must be a well-formed URL"
    )
```